src/components/view/index.tsx

```diff
@@ -142,6 +142,7 @@ function View(props: ViewProps, ref: any) {
 
   return (
     <ViewContainer
+      //@ts-ignore
```

Comment:
Is it something that might be fixed by react-native types?
Maybe use `@ts-expect-error` instead?

Reply:
Changed. Don't really know where this error comes from, might be a RN bug...